website/static/_redirects

```diff
@@ -0,0 +1,6 @@
+https://katex.netlify.com/* https://katex.org/:splat 301!
+
+/function-support.html /docs/supported.html 301
+
+/docs/en/* /docs/:splat 301
+/docs /docs/node.html 301

```

Comment:
Are redirects needed for each subdirectory?

Reply:
@kevinbarabash We don't have `index.html` for `docs` directory, so people going to https://katex.org/docs will have 404. Also previously, there were 404s due to wrong URLs (`docs/en`, #1507), so this redirects `docs/en/XXX` to `docs/XXX`.